article/RNH_AR_EG_SSTJ_EP_MMC-2016.md

```diff
@@ -120,23 +125,31 @@ $$F_{NLS} = \frac{1}{2} \sum_{m}^{i=1} \left
 - (1-f)\exp(-\sum_{j=1}^{7}W_{ij}\gamma_{j})\right ]^{2}$$ {#eq:3}
 
 Similarly to the original article [@Hoy2014-lk,], the WLS procedure is only used here
-to obtain the intial guess for the free water elimination parameters, which were then
-used to initialize the non-linear convergence procedure (see below).
+to obtain the initial guess for the free water elimination parameters, which were then
+used to initialize the non-linear solver (see below).
```

Comment:
A small style fix, nonlinear stuff is not always guaranteed to converge (even without the numerical errors it can accumulate)